sql-statements/sql-statement-pitr.md

```diff
@@ -0,0 +1,242 @@
+
+#### 语法图
+
+```ebnf+diagram
+StreamStatusStmt ::=
+    "SHOW" "BACKUP" "LOGS" "STATUS"
+```
+
+#### 示例
+
+{{< copyable "sql" >}}
+
+```sql
+SHOW BACKUP LOGS STATUS;
+```
+
+### 暂停日志备份任务
+
+等价于执行 `br log pause` 命令，暂停正在运行的日志备份任务。
+
+#### 语法图
+
+```ebnf+diagram
+StreamPauseStmt ::=
+    "PAUSE" "BACKUP" "LOGS" StreamPauseOption*
+    
+StreamPauseOption ::=
+    "GC_TTL" '='? GCTTLTSO
+
+GCTTLTSO ::=
+    LengthNum | stringLit    
+```
+
+#### 示例
+
+{{< copyable "sql" >}}
+
+```sql
+PAUSE BACKUP LOGS;
+PAUSE BACKUP LOGS GC_TTL = '2022-08-26 21:20:00+0800';
+```
+
+### 恢复日志备份任务
+
+等价于执行 `br log resume` 命令，恢复被暂停的日志备份任务。
+
+#### 语法图
+
+```ebnf+diagram
+StreamResumeStmt ::=
+    "Resume" "BACKUP" "LOGS" 
+```
+
+#### 示例
+
+{{< copyable "sql" >}}
+
+```sql
+RESUME BACKUP LOGS;
+```
+
+### 停止日志备份任务
+
+等价于执行 `br log stop` 命令，停止日志备份任务。
+
+#### 语法图
+
+```ebnf+diagram
+StreamStopStmt ::=
+    "Stop" "BACKUP" "LOGS" 
+```
+
+#### 示例
+
+{{< copyable "sql" >}}
+
+```sql
+STOP BACKUP LOGS;
+```
+
+### 查看备份数据元信息
+
+等价于执行 `br log metadata` 命令，查看备份存储中保存的日志备份的元信息，例如最早和最近的可恢复时间点。
+
+#### 语法图
+
+```ebnf+diagram
+StreamMetaDataStmt ::=
+    "SHOW" "BACKUP" "LOGS" "METADATA" "FROM" stringLit 
+```
+
+#### 示例
+
+{{< copyable "sql" >}}
+
+```sql
+SHOW BACKUP LOGS METADATA FROM 's3://backup-101/logbackup?access-key=${access-key}&secret-access-key=${secret-access-key}"';
+```
+
+### 清理日志备份数据
+
+等价于执行 `br log truncate` 命令，从备份存储中删除过期或不再需要的备份日志数据。
+
+#### 语法图
+
+```ebnf+diagram
+StreamMetaPurgeStmt ::=
+    "PURGE" "BACKUP" "LOGS" "METADATA" "FROM" stringLit StreamPurgeOption*
+    
+StreamPurgeOption ::=
+    "UNTIL_TS" '='? UNTILTSO
+
+UNTILTSO ::=
+    LengthNum | stringLit    
+```
+
+#### 示例
+
+{{< copyable "sql" >}}
+
+```sql
+PURGE BACKUP LOGS METADATA FROM 's3://backup-101/logbackup?access-key=${access-key}&secret-access-key=${secret-access-key}"';
+PURGE BACKUP LOGS METADATA FROM 's3://backup-101/logbackup?access-key=${access-key}&secret-access-key=${secret-access-key}"' UNTIL_TS = '2022-08-26 21:20:00+0800' ;
+```
+
+## 恢复到指定时间点 PITR
+
+执行 PITR 语句需要 `RESTORE_ADMIN` 或 `SUPER` 权限。此外，执行恢复的 TiDB 节点和集群中的所有 TiKV 节点都必须有对目标存储的读或写权限。
+
+```sql
+  RESTORE POINT FROM              --restore cluster to a point
+```
+
+### 恢复到指定时间点
+
+等价于执行 `br restore point` 命令，在新集群上进行 PITR，或者只恢复日志备份数据。
+
+#### 语法图
+
+```ebnf+diagram
+StreamRestorePITStmt ::=
+    "RESTORE" "POINT" "FROM" stringLit StreamRestorePITOption*
+
+StreamRestorePITOption ::=
+    "START_TS" '='? StartTSO
+|   "FULL_BACKUP_STORAGE" '='? stringLit
+|   "START_TS" '='? StartTSO
+|   "STORED_TS" '='? StoredTSO
+
+StartTSO ::=
+    LengthNum | stringLit
+    
+StoredTSO ::=
+    LengthNum | stringLit    
+```
+
+#### 示例
+
+{{< copyable "sql" >}}
+
+```sql
+RESTORE POINT FROM 's3://backup-101/logbackup?access-key=${access-key}&secret-access-key=${secret-access-key}"' FULL-BACKUP-STORAGE = 's3://backup-101/snapshot-202205120000?access-key=${access-key}&secret-access-key=${secret-access-key}"';
```

Comment:
Is here `FULL-BACKUP-STORAGE` (kebab-style) instead of `FULL_BACKUP_STORAGE`?

Reply:
Fixed